### DISCUSSION TITLE
Cross reference to a sentence in the text

### DISCUSSION THREAD

**Discussion by Santonia27** [quarto-dev/quarto-cli#7212] [Q&A]
### Description

Hi,

I would like to create cross references to sentences or sections but without the reference looking like "Sec 0.1.2.1". 
I want to replace this with my own words, such as "here" 

For this and this look "here".....

Can I change this in either the cross-ref sec settings an/or can I refer to only sentences? 

Thanks!

**Comment by cderv** [comment]
I believe with Pandoc's Markdown, you can link to headings using their id (or implicit id) and a usual link syntax  (https://pandoc.org/MANUAL.html#extension-implicit_header_references). This should work in a single document with Quarto too. 

Cross ref brings numbering and cross document support. For readers, it is best to include the numbers, and I don't think you can remove it. 
You can remove the label though but not the numbers: https://quarto.org/docs/authoring/cross-references.html#references
And you can tweak it through option : https://quarto.org/docs/authoring/cross-references.html#options

@cscheid is there a way to crossref with just a custom text ?

**Reply by Santonia27** [reply]
The first solution is already working and acceptable for my purpose. Thanks a lot!

**Reply by cscheid** [reply]
> @cscheid is there a way to crossref with just a custom text ?

Not yet, but "generic anchor crossrefs" are part of #4944. Stay tuned!
